deploy/helm/sumologic/templates/_helpers.tpl

```diff
@@ -16,11 +16,10 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- end -}}
 
 {{/*
-Create default fully qualified labels.
-We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+Create default labels
 */}}
 {{- define "sumologic.labels.app" -}}
-{{- template "sumologic.fullname" . }}
+{{- printf "collection-sumologic" }}
```

Comment:
Are there any upgrade risks with this change and someone who has used a different release/namespace? I believe labels are immutable and one thing I could see is helm upgrade requiring a `--force` flag.  If so, it would be good to document in 1.1.

Reply:
Yes, it requires `--force` flag during upgrade

```
Error: UPGRADE FAILED: StatefulSet.apps "falling-zorse-sumologic-fluentd-events" is invalid: spec: Forbidden: updates to statefulset spec for fields other than 'replicas', 'template', and 'updateStrategy' are forbidden && StatefulSet.apps "falling-zorse-sumologic-fluentd-metrics" is invalid: spec: Forbidden: updates to statefulset spec for fields other than 'replicas', 'template', and 'updateStrategy' are forbidden && StatefulSet.apps "falling-zorse-sumologic-fluentd-logs" is invalid: spec: Forbidden: updates to statefulset spec for fields other than 'replicas', 'template', and 'updateStrategy' are forbidden
```

Reply:
As the template is used for all app labels, it shouldn't affect anything (not sure about changed servicemonitors definitions)

Reply:
Let's make sure we add a release note with this change and error and flag.

Reply:
I have been created #680 to not forget about it

Reply:
👍